web/cypress/integration/route/create-route-with-search-service-and-set-priority.spec.js

```diff
@@ -0,0 +1,151 @@
+    ip: '127.0.0.1',
+    port: 8000,
+    weight: 1,
+    serviceName2: 'service_test2',
+    ip2: '127.0.0.1',
+    port2: 8000,
+    weight2: 1,
+    priority: 1,
+    createServiceSuccess: 'Create Service Successfully',
+    deleteServiceSuccess: 'Delete Service Successfully',
+    createRouteSuccess: 'Submit Successfully',
+    deleteRouteSuccess: 'Delete Route Successfully',
+    routeName: 'route_test1',
+    searchServiceName: 'service_test2',
+    upstreamName: 'None',
+  };
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('should create two services', function () {
+    cy.visit('/');
+    cy.contains('Service').click();
+    cy.contains('Create').click();
+
+    cy.get(selector.name).type(data.serviceName);
+    cy.get(selector.description).type(data.description);
+    cy.get(selector.nodes_0_host).click();
+    cy.get(selector.nodes_0_host).type(data.ip);
+    cy.get(selector.nodes_0_port).clear().type(data.port);
+    cy.get(selector.nodes_0_weight).clear().type(data.weight);
+
+    cy.contains('Next').click();
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.get(selector.notification).should('contain', data.createServiceSuccess);
+
+    cy.visit('/');
+    cy.contains('Service').click();
+    cy.contains('Create').click();
+
+    cy.get(selector.name).type(data.serviceName2);
+    cy.get(selector.description).type(data.description);
+    cy.get(selector.nodes_0_host).click();
+    cy.get(selector.nodes_0_host).type(data.ip2);
+    cy.get(selector.nodes_0_port).clear().type(data.port2);
+    cy.get(selector.nodes_0_weight).clear().type(data.weight2);
+
+    cy.contains('Next').click();
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.get(selector.notification).should('contain', data.createServiceSuccess);
+  });
+
+  it('should create route with search service and set priority', function () {
+    cy.visit('/');
+    cy.contains('Route').click();
+    cy.contains('Create').click();
+    cy.contains('Next').click().click();
+
+    // set name
+    cy.get(selector.name).type(data.routeName);
+    cy.get(selector.serviceSelector).type(data.serviceName2 + '\n');
+    // set priority
+    cy.get(selector.priority).type(data.priority);
+    cy.contains('Next').click();
+    // select upstream with None
+    cy.get(selector.upstreamSelector).type(data.upstreamName + '\n');
+    cy.contains('Next').click();
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.get(selector.result).should('contain', data.createRouteSuccess);
+  });
+
+  it('should check the priority was seted', function () {
+    cy.visit('/');
+    cy.contains('Route').click();
+    cy.get(selector.name).type(data.routeName + '\n');
+    cy.contains(data.routeName).siblings().contains('More').click();
+    cy.contains('View').click();
+    cy.get(selector.drawer).should('be.visible');
+    cy.get(selector.monacoScroll).within(() => {
+      cy.contains('priority').should('exist');
```

Comment:
LGTM

It would be better to check whether `service_id` exist

Reply:
i was add  check whether service_id exist for the test case